cmd/validators.go

```diff
@@ -102,41 +102,22 @@ func inferFromTo(src, dst string) common.FromTo {
 		return common.EFromTo.Unknown()
 	}
 
-	switch {
-	case srcLocation == common.ELocation.Local() && dstLocation == common.ELocation.Blob():
-		return common.EFromTo.LocalBlob()
-	case srcLocation == common.ELocation.Blob() && dstLocation == common.ELocation.Local():
-		return common.EFromTo.BlobLocal()
-	case srcLocation == common.ELocation.Local() && dstLocation == common.ELocation.File():
-		return common.EFromTo.LocalFile()
-	case srcLocation == common.ELocation.File() && dstLocation == common.ELocation.Local():
-		return common.EFromTo.FileLocal()
-	case srcLocation == common.ELocation.Pipe() && dstLocation == common.ELocation.Blob():
-		return common.EFromTo.PipeBlob()
-	case srcLocation == common.ELocation.Blob() && dstLocation == common.ELocation.Pipe():
-		return common.EFromTo.BlobPipe()
-	case srcLocation == common.ELocation.Local() && dstLocation == common.ELocation.BlobFS():
-		return common.EFromTo.LocalBlobFS()
-	case srcLocation == common.ELocation.BlobFS() && dstLocation == common.ELocation.Local():
-		return common.EFromTo.BlobFSLocal()
-	case srcLocation == common.ELocation.Blob() && dstLocation == common.ELocation.Blob():
-		return common.EFromTo.BlobBlob()
-	case srcLocation == common.ELocation.File() && dstLocation == common.ELocation.Blob():
-		return common.EFromTo.FileBlob()
-	case srcLocation == common.ELocation.Blob() && dstLocation == common.ELocation.File():
-		return common.EFromTo.BlobFile()
-	case srcLocation == common.ELocation.File() && dstLocation == common.ELocation.File():
-		return common.EFromTo.FileFile()
-	case srcLocation == common.ELocation.S3() && dstLocation == common.ELocation.Blob():
-		return common.EFromTo.S3Blob()
-	case srcLocation == common.ELocation.Benchmark() && dstLocation == common.ELocation.Blob():
-		return common.EFromTo.BenchmarkBlob()
-	case srcLocation == common.ELocation.Benchmark() && dstLocation == common.ELocation.File():
-		return common.EFromTo.BenchmarkFile()
-	case srcLocation == common.ELocation.Benchmark() && dstLocation == common.ELocation.BlobFS():
-		return common.EFromTo.BenchmarkBlobFS()
-	case srcLocation == common.ELocation.GCP() && dstLocation == common.ELocation.Blob():
-		return common.EFromTo.GCPBlob()
+	out := common.EFromTo.Unknown()
```

Comment:
can we maybe add a comment explaining the logic below since it's a little unintuitive